src/utils.jl

```diff
@@ -112,6 +112,13 @@ function write_data(
     return
 end
 
+#Given the changes in syntax in ParameterJuMP and the new format to create anonymous parameters
+function add_parameter(model::JuMP.Model, val::Number)
```

Comment:
@kdheepak does this count as type piracy?

Reply:
No. You own the function.

Reply:
@odow thanks.